pkg/app/piped/cmd/piped/piped.go

```diff
@@ -170,6 +170,24 @@ func (p *piped) run(ctx context.Context, input cli.Input) (runErr error) {
 		}
 	}
 
+	// Login to OCI registry.
+	if regs := cfg.OCIHelmChartRegistries(); len(regs) > 0 {
+		reg := toolregistry.DefaultRegistry()
+		helm, _, err := reg.Helm(ctx, "")
+		if err != nil {
+			return fmt.Errorf("failed to find helm to login to OCI registry (%w)", err)
+		}
+
+		for _, r := range regs {
+			if r.Username != "" && r.Password != "" {
+				err := loginToOCIRegistry(ctx, helm, r.Address, r.Username, r.Password)
```

Comment:
Does this login give a permanent session or a temporary session?
I wonder about the behavior for long-running Piped, since this login is run once when startinging up.

Reply:
After executing `helm registry login`, the login context will be stored on a dedicated helm config file (e.g. `/path/to/registry/config.json`, it's can be changed with the `HELM_REGISTRY_CONFIG` envvar) so the session is permanent unless that config file is deleted or modified.

Reply:
@Szarny Thank you for your explanation. That is great. So there will be no problem with that.